src/main/java/nl/workingtalent/wtacademy/loan/ReadLoanDto.java

```diff
@@ -13,6 +13,8 @@ public class ReadLoanDto {
 	private LocalDate endDate;
 	private State conditionStart;
 	private State conditionEnd;
+	private String bookTitle;
+	private List<String> bookAuthors;
 	private long userId;
 	private Long reservationId;
 	private Long bookCopyId;
```

Comment:
De book id is ook nog handig om mee te sturen, zo kan je als je op de title klikt de id gebruiken om naar de juiste book details pagina te gaan.  Alex heeft dit al toegevoegd op zijn reservering goedkeuren/afwijzen branch.